Example/Utils/Config.swift

```diff
@@ -8,30 +8,46 @@ class Config: NSObject {
              version                             = "CFBundleShortVersionString",
              subscriptionKey                     = "RASProjectSubscriptionKey",
              endpoint                            = "RMAAPIEndpoint",
+             stagingProjectId                    = "RASStagingProjectId",
+             stagingSubscriptionKey              = "RASStagingProjectSubscriptionKey",
+             stagingEndpoint                     = "RMAAPIStagingEndpoint",
              isPreviewMode                       = "RMAIsPreviewMode",
              requireMiniAppSignatureVerification = "RMARequireMiniAppSignatureVerification",
              sslKeyHash                          = "RMASSLKeyHash"
     }
 
-    static let userDefaults = UserDefaults(suiteName: "com.rakuten.tech.mobile.miniapp.MiniAppDemo.settings")
+    private enum InternalKey: String {
+        case stagingActive = "MiniApp.Settings.StagingEnvironmentActive"
+    }
+
+    static var isProd: Bool {
+        !UserDefaults.standard.bool(forKey: InternalKey.stagingActive.rawValue)
```

Comment:
This might crash at first launch

```suggestion
        if let prod = UserDefaults.standard.bool(forKey: InternalKey.stagingActive.rawValue) {
            return prod
        } else {
            return false
        }
```

Reply:
@Climbatize 
Can you explain more in detail why it should crash? 
UserDefaults returns false for a non existing key so it behaves as expected using prod first. I've tested couple of times with deleting the app first.

Reply:
Ohhhhhh my bad !!! Friday review... I read a force unroll instead of a boolean manipulation 🤦‍♂️